specification/riskiq/Easm/models.tsp

```diff
@@ -1351,6 +1366,7 @@ enum ReportBillableAssetBreakdownKind {
 
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "Documentation will be added next preview version"
 @doc("The state the task is in.")
+@projectedName("csharp", "TaskResourceState")
```

Comment:
You'll want to use `@clientName` in the `client.tsp` here . `@projectedName` is in the process of being deprecated and we are changing all instances

Reply:
(same for all other `@projectedName`)

Reply:
@timotheeguerin I'm getting this error when running npx tsv command: 
<img width="1273" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/127871101/ed1b4da0-25e5-4984-8c52-ec259dc1b5d9">
I'm using clientName like this: 
<img width="338" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/127871101/d3f7e5d6-1411-4204-b657-acbf8458e8b2">
Is there any way that I can fix without changing the name of model?

Reply:
Also I think they want to keep tcgc decorator in the client.tsp with augment.

```
@@clientName(Source, "SourceDetails", "csharp")
```

Reply:
Typespec client generator core